profiles/ZahraShahid.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/65255043/193467864-9c1d9364-b086-4e64-95bc-c46d68a5ef48.png",
+  "issueId": 1450,
+  "name": "Zahra Shahid",
+  "username": "ZahraShahid"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

Resolved